package.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>gz-math7</name>
+  <version>7.4.0</version>
+  <description>Gazebo Math : Math classes and functions for robot applications</description>
+  <maintainer email="natekoenig@gmail.com">Nate Koenig</maintainer>
```

Comment:
@adityapande-1995 and I are listed as the [CODEOWNERS](https://github.com/gazebosim/gz-math/blob/gz-math7/.github/CODEOWNERS)

Reply:
I copied this from the README. Should we update that as well?

Reply:
yeah, let's update the README as well

Reply:
or remove it from the README

Reply:
I guess we list maintainers in the README of other packages

Reply:
so yes, let's update the readme as well

Reply:
Updated in https://github.com/gazebosim/gz-math/pull/581/commits/3054479a4a60d35c02d0c617950df6076a49af24